src/main/kotlin/app/revanced/patches/reddit/customclients/relayforreddit/api/fingerprints/GetLoggedOutBearerTokenFingerprint.kt

```diff
@@ -0,0 +1,7 @@
+package app.revanced.patches.reddit.customclients.relayforreddit.api.fingerprints
+
+object GetLoggedOutBearerTokenFingerprint :
+    AbstractClientIdFingerprint(
+        "RedditAccountManager;",
+        "https://oauth.reddit.com/grants/installed_client"
```

Comment:
Are really both strings required?

Reply:
Yes, one is the name of the class, the other is a string from the method, and there are multiple methods in the same class that need to be patched

Reply:
I mean it's likley possible to skip the class name, since it's unlikley that any other method contains https://oauth.reddit.com/grants/installed_client. But I still think it makes more sense to keep it, so that it is consistent with the the other fingerprints (which most likely need both strings)